frontend/packages/kubevirt-plugin/src/topology/TopologyVmDetailsPanel.tsx

```diff
@@ -28,9 +28,8 @@ type LoadedTopologyVmDetailsPanelProps = TopologyVmDetailsPanelProps & {
 const LoadedTopologyVmDetailsPanel: React.FC<LoadedTopologyVmDetailsPanelProps> = observer(
   ({ loaded, vmNode, templates }) => {
     const vmData = vmNode.getData();
-    const { pod } = vmData.data.vmStatusBundle;
     const vmObj = vmData.resource as VMKind;
-    const { vmi, vmStatusBundle } = vmData.data;
+    const { vmi, vmStatusBundle, pods = [] } = vmData.data;
```

Comment:
Could we use `usePodsForVm` here rather than carrying pods around in the vmData?

Reply:
Hi Jeff :)

Im guessing i could use something like this:

    `const { podData: { pods = [] } = {} } = usePodsForVm(vmi);`

I want aware of this custom hook